model/src/pyrenew/process/rtrandomwalk.py

```diff
@@ -3,7 +3,7 @@
 
 import numpyro as npro
 import numpyro.distributions as dist
-from numpyro.distributions import transforms as nt
+import pyrenew.transformation as nt
```

Comment:
Should it be just `t` or `pt` now that it's our own module?

Reply:
Fixed here ef26272.